angular-app/src/app/app.module.ts

```diff
@@ -170,7 +167,7 @@ import { AdminPanelWidgetComponent } from './theme/widgets/admin-panel-widget/ad
     DateManager,
     ScheduleService,
     UserService,
-    AppointmentService
```

Comment:
Why is `AppointmentService` being removed from the declarations array? Perhaps this wasn't intentional?

Reply:
It's already in the `providers` array on line 165, so I removed the second one. I think we had two different versions of `AppointmentService`, and a merge just left both of them there.

Reply:
Ah gotcha, did not catch that it was already there.